rust/agents/scraper/src/scraper/block_cursor.rs

```diff
@@ -0,0 +1,91 @@
+use std::time::{Duration, Instant};
+
+use eyre::Result;
+use sea_orm::prelude::*;
+use sea_orm::{ActiveValue, Insert, Order, QueryOrder, QuerySelect};
+use tokio::sync::RwLock;
+use tracing::{debug, instrument, trace, warn};
+
+use crate::date_time;
+use crate::db::cursor;
+
+const MAX_WRITE_BACK_FREQUENCY: Duration = Duration::from_secs(10);
```

Comment:
Frequency is measured in `1/<unit time>`, and this is the max `saves/s` so I think it is logical. I mean all this does is make sure it can't write _too frequently_ to the database. I mean `MAX_WRITE_BACK_RATE` would also be right it just feels a little less obvious to me.

Reply:
ok, happy to keep as is